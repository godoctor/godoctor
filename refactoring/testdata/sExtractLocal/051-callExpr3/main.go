package main

import "fmt"

func main() {
	fmt.Println("Hello, 世界") // <<<<< extractLocal,6,14,6,28,newVar,pass
}
