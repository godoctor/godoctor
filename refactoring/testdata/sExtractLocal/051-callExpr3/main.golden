package main

import "fmt"

func main() {
	newVar := "Hello, 世界"
	fmt.Println(newVar) // <<<<< extractLocal,6,14,6,28,newVar,pass
}
